### DISCUSSION TITLE
[ARCHIVED] Как поживает методология в контексте фреймворков?

### DISCUSSION THREAD

**Discussion by azinit** [feature-sliced/documentation#32] [Archive]
Большинство энтузиастов feature-sliced подхода (в той или иной интерпретации), чаще всего юзают React в своих проектах

И он действительно хорош и в том плане, что позоляет крутить архитектуру и структуру файлов - как нам угодно

### Но не стоит забывать и про полноценные фреймворки аля `Vue`, `Angular`

Отсюда интересно - пробовал ли кто-то  (или мб видел) в том или ином виде `feature-based` подход вне React?

> P.S. @niyazm524 думаю тут ты можешь зажечь)
> 
> Но само собой - приветствуются любые мнения

**Comment by ilyaagarkov** [comment]
Чисто теоретически я не вижу проблем данного подхода и для нативных приложений на Swift/Flutter и прочих. 
Так как я вижу feature-sliced как методологию для построения интерфейсов.

Единственное отличие это папка `pages`. Так как нативные приложения, оперируют термином sсreen, и у них совсем другие взаимоотношения между собой по сравнению с web

**Comment by azinit** [comment]
Особенно интересно, если кто-то распишет опыт на `NextJS` и `Angular`)
